tutorials/tutorial_1/containers/prediction-model/Dockerfile

```diff
@@ -1,38 +1,64 @@
-# !! EDIT THIS: Set the version of Python to be used.
-FROM python:3.8-bullseye as wheelbuilder
+# !! EDIT THIS: Set base Docker image to be used.
```

Comment:
Not sure why we had the "EDIT THIS" comment here - we don't actually expect users to edit this doc...
```suggestion
# Set base Docker image to be used.
```